nerfstudio/viewer/server/viewer_utils.py

```diff
@@ -274,6 +274,15 @@ def __init__(self, config: cfg.ViewerConfig, log_filename: Path):
         self.webrtc_thread = None
         self.kill_webrtc_signal = False
 
+    def _pick_drawn_image_idxs(self, total_num: int) -> list[int]:
+        """Determine indicies of images to display in viewer."""
```

Comment:
Can you add `Args: ...` and `Returns` to the docstring.

Reply:
Sure!